### PR TITLE
fix #45540 macsoft.jp

### LINKED ISSUE #45540: macsoft.jp
### Issue URL (Ads)
[http://www.macsoft.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.macsoft.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 1.5.4
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#45540

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@ hosyusokuhou.jp#?##side > div.sidewrapper:has(> div.sidetitle:contains(★スポ
 ! https://github.com/AdguardTeam/AdguardFilters/issues/40218
 ! https://github.com/AdguardTeam/AdguardFilters/issues/23716
 ascii.jp#?#body > #jpWrap > #header:has(> .adsize728)
+! https://github.com/AdguardTeam/AdguardFilters/issues/45540
+macsoft.jp#?#div.neko:has( > a[href*="jirokichi.co.jp/products/kaikei_jirou"] > img[src="/img/Banner1/kaikei_banner.gif"])
```
Comment (BlazDT) [inline JapaneseFilter/sections/css_extended.txt]: Some filters block via url the word banner, so better drop the > img part.

File: JapaneseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -8,6 +8,10 @@ hosyusokuhou.jp#?##side > div.sidewrapper:has(> div.sidetitle:contains(★スポ
...
+macsoft.jp#?#table:has( > tbody > tr > td > div.K12:contains(Parallels Desktop 12 for Mac))
+macsoft.jp#?#td:has( > ins.adsbygoogle)
```
Comment (BlazDT) [inline JapaneseFilter/sections/css_extended.txt]: Only removes a very small part, so I do not think we need this rule.

**Review by BlazDT — CHANGES_REQUESTED** [review]
Unsure if we should block this one too (with safer ext-CSS rule):
td[align="center"] > .flame[width="230px"]

**Comment by Sergey-Lyapin** [comment]
> Unsure if we should block this one too (with safer ext-CSS rule):
> td[align="center"] > .flame[width="230px"]

there are obsolete links to the apple shop, i will add this rule, thanks

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -10,6 +10,7 @@ javkun.com##.adsvideo
 69re.tv###wrapper > div.container > div[style="width: 100%"] > a[target="_blank"] > img
 hosyusokuhou.jp###side > div.sidewrapper[style^="width: 218px; margin-bottom: 0px; position:"] a[target="_blank"] > img
 hosyusokuhou.jp##.abanner
+macsoft.jp##td[align="center"] > .flame[width="230px"]
```
Comment (BlazDT) [inline JapaneseFilter/sections/specific.txt]: Please use an ext-CSS rule (which I described in previous comment). Site is using .flame for valid content too and only [width] selector is too generic.

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**